test/widgets/inbox_test.dart

```diff
@@ -51,6 +51,11 @@ void main() {
     await tester.pumpAndSettle();
   }
 
+  List<StreamMessage> generateStreamMessages(ZulipStream stream, int count) {
+    return List.generate(count, (index) => eg.streamMessage(
+      stream: stream, topic: '${stream.name} topic $index', flags: []));
```

Comment:
Added the `flags` just in here, not in the `generateStreamMessages` parms. Should that be added in the `params`? Because it would still not give the full control to specify the `flags` for each individual `message`!

Reply:
Yeah, let's have `generateStreamMessages` take this as a parameter, so that `prepareVarious` can say `flags: []` explicitly (the same way it does for its direct calls to `eg.streamMessage`).

It's fine that the flags will be the same for all the messages in a given `generateStreamMessages` call. We don't currently have a use case for something more complex; if in the future we do, we can come up with an appropriate API for that then.